addons/udes_load_testing/tests/common.py

```diff
@@ -134,6 +100,18 @@ def test_report(self):
             for line in lines:
                 print(line)
 
+
+
+class BackgroundDataRunner(LoadRunner):
+    _N = int(1e5)
+    _dummy_picking_type = None
```

Comment:
This is an unusual style to use in Python. Just initialize them in the setUpClass method you have already.